src/styles/components/card.scss

```diff
@@ -0,0 +1,106 @@
+$card-width: fit-content;
+$card-bg-color: $basic-color;
+$card-font-size-basic: $font-size-basic;
+$card-fw-bold: 700;
+$card-fw-up-medium: 500;
+$card-padding: 16px;
+$card-parts-width: 166px;
+$card-parts-flex-direction: column;
+$card-space-between-elements: $card-font-size-basic + 2;
+$card-border: 1px solid #f3f3f3;
+$link-height: 40px;
+$card-link-border-rad: 5px;
+$card-link-first-color: $secondary-color;
+$card-link-second-color: $basic-color;
```

Comment:
it's good when you try to use variables, but no needs to add so many variables and then make calculations because it makes your code hard to read 
```suggestion
```suggestion
$card-width: fit-content; // delete - used only once
$card-bg-color: $basic-color; // delete - you have $basic-color
$card-font-size-basic: $font-size-basic; // delete - you have $font-size-basic
$card-fw-bold: 700; // delete or can be moved to common variables as $fw-bold: 700;
$card-fw-up-medium: 500; // delete or can be moved to common variables as $fw-medim 500;
$card-padding: 16px; // delete - because you should have padding: 32px 16px 16px;
$card-parts-width: 166px; // delete - because it's a bad practice to limit width like this. And remove hardcoding width from this element
$card-parts-flex-direction: column; // delete - used  only once
$card-space-between-elements: $card-font-size-basic + 2; // delete - because you literally have different indent between elements
$card-border: 1px solid #f3f3f3; // it's ok
$link-height: 40px; // delete - it's not good practice to hardcode height for elements with content inside. Just use padding for your links
$card-link-border-rad: 5px; // delete - just redundant
$card-link-first-color: $secondary-color;  // delete - you have $secondary-color
$card-link-second-color: $basic-color; // delete - you have $basic-color
```

Please apply such recommendations for other files as well